Assets/Scripts/Abilities.cs

```diff
@@ -214,4 +214,328 @@ private IEnumerator _flashlightAbility(PlayerCharacterStats stats)
             EnableButtons();
         }
     }
+
+
+	public void CoveringFire(PlayerCharacterStats stats) { if (stats.Actionsleft > 0) StartCoroutine(_CoveringFire(stats)); }
+	private IEnumerator _CoveringFire(PlayerCharacterStats stats)
+	{
+		try {
+			DisableButtons();
+			var abilData = (from abil in stats.AbilityData where abil.Name == "CoveringFire" select abil).FirstOrDefault();
+			//Debug.Log(abilData.Description);
+			GetComponent<PlayerMovementManager>().SetQuadsEnabled(false);
+			GetComponent<PlayerMovementManager>().enabled = false;
+			do
+			{
+				if (Input.GetKeyDown(KeyCode.Escape)) yield break;
+				yield return null;
+			} while (!Input.GetMouseButtonDown(0));
+			stats.isCoveringFire = true;
+			abilData.Currcooldown = abilData.Maxcooldown;
+			//stats.hasMoved = true;
+			stats.Actionsleft--;
+			GetComponent<TurnManager>().AutoEndTurnCheck();
+			yield return new WaitForSeconds(.5f);
+		}
+		finally {
+			GetComponent<PlayerMovementManager>().enabled = true;
+			if (stats.Actionsleft == 0)
+				GetComponent<PlayerMovementManager>().Deselect ();
+			else
+				GetComponent<PlayerMovementManager>().Select (stats);
+			EnableButtons();
+
+		}
+			
+	}
+
+	public void Fortify(PlayerCharacterStats stats) { if (stats.Actionsleft > 0) StartCoroutine(_Fortify(stats)); }
+	private IEnumerator _Fortify(PlayerCharacterStats stats)
+	{
+		try {
+			DisableButtons();
+			var abilData = (from abil in stats.AbilityData where abil.Name == "Fortify" select abil).FirstOrDefault();
+			//Debug.Log(abilData.Description);
+			GetComponent<PlayerMovementManager>().SetQuadsEnabled(false);
+			GetComponent<PlayerMovementManager>().enabled = false;
+			do
+			{
+				if (Input.GetKeyDown(KeyCode.Escape)) yield break;
+				yield return null;
+			} while (!Input.GetMouseButtonDown(0));
+
+			stats.MitigationValue = 2;
```

Comment:
This will never wear off.
Also, why are you just setting it to 2?

Reply:
I used the 2 as a quick test when making it, I thought the change to making based of ability data was in my push.  I'll make sure it is changed and fix it to wear off